p2p/libp2p/protocol/protocol.go

```diff
@@ -0,0 +1,9 @@
+package protocol
+
+import "github.com/libp2p/go-libp2p/core/protocol"
+
+// PoktProtocolID is the libp2p protocol ID matching current version of the pokt protocol.
+var PoktProtocolID = protocol.ID("pokt/v1.0.0")
+
+// DefaultTopicStr is a "default" pubsub topic string for use when subscribing.
+var DefaultTopicStr = "pokt/default"
```

Comment:
1. Why not constant?
2. Does this need to be exported (i.e. capital)
3. Use factoring to declare all the vars with one `var` or `const` statement

Reply:
1. My thinking was that perhaps we would want to set these with an `LDFLAGS=-X ...` at compile-time, which I thought required them to be `var`s but 1) I could be mistaken about that (see: [go tool link --help](https://pkg.go.dev/cmd/link)) and 2) don't have a strong opinion about it at this point.
2. This is all there is to the `protocol` package currently so to use it anywhere else, it would have to be exported, right? The only place it's currently used is in the libp2p module, are you suggesting to move it there?
3. :+1: thanks for calling that out!

Reply:
@Olshansk did you have any more input on this point?

Reply:
1. My opinion is that iff we ever up needing it to be an `LDFLAG`, we can convert it to a var then, and make the flag is being used. Otherwise, please add a comment that says "this is a var and not a const because we might need it to be a flag one day".

2. I'm suggesting that you s/DefaultTopicStr/defaultTopicStr so that the developer knows "this is only being used in the context of this package"

3. 👍

Reply:
We can continue the discussion in the relevant follow-on PR but I just wanted to clarify on point 2  to make sure we weren't talking past each other. Unless these are moved to the package where they're being used, they must be exported, no? 

I think moving them would be a reasonable suggestion; although, `PoktProtocolID` is being used in `module` and `network`, as currently written, so I think it must be exported. I anticipate that use of these by multiple packages would likely continue in the work to follow. While I agree that we should prefer to export things as necessary, in this case, for `DefaultTopicStr` I think keeping it close to `PoktProtocolID` improves maintainability and readability than moving it would.